package.json

```diff
@@ -28,6 +27,7 @@
   ],
   "devDependencies": {
     "chai": "3.5.0",
+    "flow-bin": "0.33.0",
```

Comment:
Why does this need to be a dev dependency if it's a peer dependency? Is it because peer dependencies aren't installed automatically? If so, can we make it more lenient like how `^0.33.0` was?